src/framework/shortcuts/view/shortcutsmodel.cpp

```diff
@@ -188,6 +188,15 @@ QVariant ShortcutsModel::currentShortcut() const
     return shortcutToObject(sc);
 }
 
+QString ShortcutsModel::getCategory(ui::ActionCategory category) const
+{
+    if ((int)category + 1 >= 0 && (int)category + 1 < categories.size()) {
+        return categories[(int)category + 1].qTranslated();
+    }
```

Comment:
It might be better to do this method as a switch statement: 
```c++
switch (category) {
case ...: return ...;
...
}

// Unreachable
return {};
```
This way, you don't need to worry about syncing the order in the enum and in this method, and you'll get a compiler warning here if you forget to handle new cases here.

Also, maybe better rename the method to `categoryName(...)` (we tend not to use `get` prefixes).

Reply:
@cbjeukendrup I can do this however, the categories vector is required to send all sections to QML as well. It is in `ShortcutsPage.qml`
```qml
  Component.onCompleted: {
      shortcutsModel.load()

      shortcutsView.allSections = shortcutsModel.sections()
      topPanel.setSearchText(root.shortcutCodeKey)
  }
```

So I thought maybe the get category could just use this list itself to preserve Lines of Code. Please let me know if I should still go with the switch case approach in section name